packages/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -489,30 +499,33 @@ $timepicker-min-width: 24rem;
 
     .k-popover-content {
       .range-display {
-        margin: var(--spacing-xs) auto var(--spacing-xs) $margin;
+        margin: 0 auto 0;
       }
       .timeframe-section {
         .timeframe-section-title {
-          margin-left: $margin;
+          font-weight: 600;
           margin-bottom: var(--spacing-xs);
         }
         .timeframe-buttons {
           flex-wrap: wrap;
+
           .timeframe-btn {
             font-size: var(--type-sm);
-            flex-basis: calc(32% - $margin);
             font-weight: 400;
-            margin: $margin;
+            // Only two columns of the 3 will have a right margin
+            flex: 0 calc(33% - ($margin/2));
```

Comment:
You won't be able to use the slash `/` for division when we upgrade Sass or on the `beta` branch. You need to use `sass:math` or just use another variable that is explicitly set to half of the value of `$margin`

Reply:
Good to know!

Reply:
I guess I'm actually asking to change it here if you don't mind